README.md

```diff
@@ -108,7 +108,7 @@ final class TwoDollars implements Money
 ```
 
 You are free to use those objects in any out-of-the-box or custom operations.
-Let sea the next example: you need some tax which is `10%` of given amount but not less than `$2`.
+Let see the next example: you need some tax which is `10%` of the given amount but not less than `$2`.
```

Comment:
this will be correct - Let's take a look at the following example:

Reply:
vote for `Let's take a look`